tbconnect/migrations/0001_initial.py

```diff
@@ -1,12 +1,11 @@
-# Generated by Django 3.1 on 2020-08-25 14:07
+# Generated by Django 3.1.14 on 2022-06-13 07:12
```

Comment:
hmm, why is there a change in migrations that have already run?

Reply:
My local project broke then i had to fix it by deleting migrations then the new initial file was created.

Reply:
We'll need to try fix this because this as is will fail in qa/prod or for any existing db.

Reply:
My suggestion is to either revert that commit if it exists or manually make this file match what's in `develop` and then re-run `makemigrations` to get a new migration for your model additions.